### PR TITLE
feat: [sf,hs] batch gets

### PR DESCRIPTION
**Pull request by manas-vessel**
## Description

_Detailed description of the changes and the intent behind them_

## Checklist

- [ ] Changes are covered by tests if behavior has been changed or added
- [ ] Tests have 100% coverage
- [ ] If code changes were made, the version in `package.json` has been bumped (matching semver)

## Resolves

If the PR resolves an open issue tag it here. For example: `Resolves #34`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@vesselapi/integrations",
-  "version": "0.0.97",
+  "version": "0.0.98",
```
Comment (averyyip) [inline package.json:3]: you'll probably need to update the version depending on what goes in

File: src/platforms/hubspot/client.ts [hunk 2]
```diff
@@ -173,16 +173,23 @@ const makeClient = () => {
...
+    request(
+      ({
+        ids,
+        after,
+        limit = HUBSPOT_MAX_PAGE_SIZE,
+      }: BatchReadObjectInput) => ({
+        url: `/crm/${API_VERSION}/${module}/batch/read`,
+        method: 'POST',
+        json: {
+          properties: properties ?? null,
```
Comment (averyyip) [inline src/platforms/hubspot/client.ts:185]: How will we fetch all properties this work when we add $native?

Comment (averyyip) [reply src/platforms/hubspot/client.ts:185]: Actually it looks like a POST request which might mean we might not run into a limit on the number of props

**Review by averyyip — DISMISSED** [review]

File: src/platforms/hubspot/client.ts (hunk 2 above)
Comment (averyyip) [reply src/platforms/hubspot/client.ts:185]: Can you create a ticket to track adding $snative to batch gets?

Comment (manas-vessel) [reply src/platforms/hubspot/client.ts:185]: I actually haven't implemented it yet for the list endpoint either. Though that will probably happen within the unification where we fetch all properties and pass it through here. So allowing additional properties to be passed through might require a separate PR

Comment (averyyip) [reply src/platforms/hubspot/client.ts:185]: Sounds good, can we make sure to track all of this work in linear?

**Review by averyyip — APPROVED** [review]

**Pull request merged**